app/views/groups/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<section class="container">
+
+    <%= form_with model: @group do |form| %>
+      <div class="my-3">
+        <%= form.text_field :name, class: "form-control w-100", placeholder: "Name" %>
+      </div>
+  
+      <div class="my-3">
+        <%= form.text_field :icon, class: "form-control w-100", placeholder: "Icon" %>
+      </div>
+  
+      <div class="text-center m-3">
+        <%= form.submit "Save", class: "btn btn-primary" %>
+      </div>
+  
+    <% end %>
+  
+  </section>
```

Comment:
- [ ] There should be an arrow back button on the new category page that takes users to the home page. Kindly add it.

  ![image](https://github.com/ahmedeid6842/Budget_App/assets/36542986/69c39499-ca50-4e70-81ea-5ab66c3f4361)

Reply:
Fixed